.github/workflows/integration.yml

```diff
@@ -89,6 +89,7 @@ jobs:
             snowflake:
               - 'dbt/**'
               - 'tests/**'
+              - 'dev-requirements.txt'
```

Comment:
So will this effectively trigger a full integration test matrix if we update `dev-requirements.txt`?

Reply:
Yep I've already merged this into two other adapter repos